README.md

```diff
@@ -6,27 +6,61 @@
 2. Practice updating state in response to an XHR payload
 3. Practice passing props among components to handle updates
 
-## Overview
+## Animal Shelter
+![Best Friends](https://media.giphy.com/media/xTiTnz5OOUn49wKbg4/giphy.gif)
 
-This is students' first big React lab. We want them to practice not only using
-props and state to manage application state, but we want them to be sending form
-data off to an API (we can either provide or walk them through a basic Express
-server with in-memory data) and updating the application state based on the
-response. It can all be pretty basic user-creation stuff -- nothing complicated
--- but we want to make sure that they get a lot of experience with different
-parts of the component lifecycle.
+Time to put all of our hard-earned knowledge to the test! This lab is fairly big, and will require you to use everything
+you've learned up to this point. Don't be intimidated, there are plenty of tests to guide you along the way! In this lab,
+we'll be working on a front-end for an animal shelter. Sadly, there still are way too many cute pets without any owners.
+Let's help them out by creating a UI in React!
 
-This lab provides good opportunities for introducing `shouldComponentUpdate()`
-and `componentDidMount()` (for initialization) for example.
+On a high level, you will be working on several components that form the UI of the animal shelter adoption application.
+Users can filter for pets by type, and can adopt a pet of their choosing. Once a pet is adopted, they cannot un-adopt
+it. No backsies!
 
-I leave the narrative domain up to you, but the main thing is having students
-write actions that use `fetch()` to call an API, and that then use `setState()`
-to render the response (and pass parts of it down as props as necessary).
+There are several components that need your attention. All of these components can be found in the `components/` folder.
+Starting with the smallest one and working our way up:
 
-This lab is meant to be pretty challenging.
+### `Pet`
+1. Should have a `pet` prop. Use the attributes in this data to render the pet card correctly. It
+should show the pet's `name`, `type`, `age` and `weight`. Based on the pet's `type`, the component also needs to contain
+either a male (`♂`) or female (`♀`) symbol.
+2. Should have an `isAdopted` prop. Using this prop, render the correct button in the pet's card: if
+the pet is adopted, show the disabled button. Otherwise, show the primary button to adopt the pet.
```

Comment:
Should probably use the semicolon here since the colon used before. So: "Using this prop, render...; otherwise, show..." (In this case, the colon indicates that the following sentence is an explanation of what came before it. So it's easier for the reader to parse the explanation when they are part of the same sentence, jointed by the semicolon.)